src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        return new Ball(new ColorSupplier().getRandomColor(), randomNumber);
```

Comment:
move random and color supplier to the class constant

and 100 is magic number, make it a constant also read more about it in the checklist for the task